### DISCUSSION TITLE
Tailwind CSS Compatibility

### DISCUSSION THREAD

**Discussion by johnnytranamp** [caplin/FlexLayout#360] [Q&A] [answered]
Was wondering if this library is compatible with Tailwind CSS? Has anyone ran into any issues?

**Comment by johnnytranamp** [comment]
I've installed Tailwind and although the overall font of my App changed, I can't seem to have my components render correctly with the Tailwind styling syntax of using (className="  "). Below I'm just trying to see if it'll render an underlined text but doesn't seem to work, any tips would be greatly appreciated! 

<img width="225" alt="tailwind" src="https://user-images.githubusercontent.com/118769555/206535477-6be8d7db-94b9-4d2c-ac4a-8af5622d51d2.PNG">
<img width="630" alt="tailwindtab" src="https://user-images.githubusercontent.com/118769555/206535482-a3481b9c-c3a4-427b-a4a3-4899b8858772.PNG">

**Comment by Ahmed-Rokenddine** [comment, marked as answer]
This Maybe Cause tailwind didn't link well with your page , try using liking tailwinde library instead of installing it , i know it's not the best way but it's still effective on mobile projects :
https://tailwindcss.com/docs/installation/play-cdn

**Comment by johnnytranamp** [comment]
I was able to figure it out! I just had to add the path of the components to the tailwind.config.cjs in order for it to render with flex layout. Thanks for the alt solution!
